public_html/wp-content/plugins/pattern-creator/src/plugins/save-post-modifier.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { dispatch, select } from '@wordpress/data';
+import { store } from '@wordpress/editor';
+import { useEffect, useState } from '@wordpress/element';
+import { registerPlugin } from '@wordpress/plugins';
+
+window.gutenbergSavePost = dispatch( store ).savePost;
```

Comment:
Why save this as a global on window, rather than a local const or something?

Reply:
For some reason, Gutenberg will re-render plugins. I think it's because parent components are unmounted. So if they reload, when it _re-stashes_ the function, it's already the one we replaced the default with and we never have a copy of the original function.